src/lib.rs

```diff
@@ -140,6 +174,31 @@ impl Index {
         }
     }
 
+    /// Create a new index with the provided fields for the given
+    /// [`Language`](lang/enum.Language.html).
+    pub fn new_for_language<I>(lang: Language, fields: I) -> Self
```

Comment:
I believe it's more idiomatic to call it `with_language` but I might be wrong

Reply:
Sure. That was my second choice, and I suppose you're right.